dataEmbedder.py

```diff
@@ -45,7 +46,10 @@ def save_config(self, config: str = EMBED_CONFIG_PATH):
 
         :param config: 配置文件路径字符串
         """
-        pass  # pass为占位符，实现时删掉该行
+        with open(config,"a") as f:
+            f.write(self.key)
+            f.write("\r")
```

Comment:
文件中为什么要保存多个密钥？使用的时候选哪个？
另外我把密钥格式改成了bytes，文件也要以字节模式打开。
```suggestion
        with open(config, 'wb') as f:
            f.write(self.key)
```